### PR TITLE
Fixed an issue with the Iterate connector returning At least one param_value references an invalid field for the 'update' request of user

### PR DESCRIPTION
**Pull request by MarcGEthyca**
Closes #<issue>

### Description Of Changes

_Fixed a typo within the Iterate connector configuration_

### Code Changes

* [ ] _Corrected iterate_config.yml_

### Steps to Confirm

* [ ] _Ensure erasure test pass_

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* Documentation:
n/a 
* [X] Issue Requirements are Met
* [X] Relevant Follow-Up Issues Created

### CODE REVIEW

**Review by SteveDMurphy — APPROVED** [review]
Thanks again @MarcGEthyca !

File: CHANGELOG.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ The types of changes are:
 - Fix an error where a connector response value of None causes a DSR failure due to a missing value [#4483](https://github.com/ethyca/fides/pull/4483)
 - Fixed system name being non-editable when locked for GVL [#4475](https://github.com/ethyca/fides/pull/4475)
 - Fixed a bug with "null" values for retention period field on data uses [#4487](https://github.com/ethyca/fides/pull/4487)
-- Fixed an issue with the Iterate connector returning At least one param_value references an invalid field for the 'update' request of user [#4514]
+- Fixed an issue with the Iterate connector returning At least one param_value references an invalid field for the 'update' request of user [#4528]
```
Comment (galvana) [inline CHANGELOG.md:60]: This needs to be moved to the `Unreleased` section, and make sure to add the rest of the link
```
[#4528](https://github.com/ethyca/fides/pull/4528)
```

File: data/saas/config/iterate_config.yml
Comment (galvana) [inline data/saas/config/iterate_config.yml:1]: The config needs a version bump so it can auto-update any existing configs.

**Review by galvana — CHANGES_REQUESTED** [review]
Outlined a few changes

File: CHANGELOG.md (hunk 1 above)
Comment (MarcGEthyca) [reply CHANGELOG.md]: Thanks! Fixed this.

File: data/saas/config/iterate_config.yml
Comment (MarcGEthyca) [reply data/saas/config/iterate_config.yml:1]: I incremented the version to 0.1.2 - from 0.1.1.

**Review by galvana — APPROVED** [review]

**Pull request merged**